core/src/main/scala/org/apache/spark/internal/io/HadoopMapReduceCommitProtocol.scala

```diff
@@ -57,6 +60,15 @@ class HadoopMapReduceCommitProtocol(jobId: String, path: String)
    */
   private def absPathStagingDir: Path = new Path(path, "_temporary-" + jobId)
 
+  /**
+   * Checks whether there are files to be committed to an absolute output location.
+   *
+   * As the committing and aborting the job occurs on driver where `addedAbsPathFiles` is always
+   * null, it is necessary to check whether the output path is specified, that may not be the case
+   * for committers not writing to distributed file systems.
```

Comment:
This also has a grammar issue. It is not clear too

Reply:
How about :
"As committing and aborting a job occurs on driver, where `addedAbsPathFiles` is always null, it is necessary to check whether the output path is specified. Output path need not be relevant for committers not writing to distributed file systems"

Reply:
This is much better.

Reply:
Thanks a lot, guys! I've just updated the comment